src/primitives/transaction.h

```diff
@@ -20,6 +20,8 @@
 #include "zcash/JoinSplit.hpp"
 #include "zcash/Proof.hpp"
 
+#include <iostream>
```

Comment:
Left over from debugging?

Reply:
Ha.. you got me... but this is a WIP :-)